frontend/www/js/omegaup/components/common/Footer.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <div class="common-footer">
+    <div class="footer-navigation">
+      <div class="footer-brand">
+        <img class="footer-logo"
+             src="/media/logo-main-white.svg">
+        <div class="slogan">
+          {{ T.frontPageFooter }}
+        </div>
+      </div>
+      <div class="footer-list-section">
+        <h4 class="section-title">{{ T.frontPageFooterSite }}</h4>
+        <ul>
+          <li>
+            <a href="/arena/">{{ T.navArena }}</a>
+          </li>
+          <li>
+            <a href="/problem/">{{ T.navProblems }}</a>
+          </li>
+          <li>
+            <a href="/rank/">{{ T.navRanking }}</a>
+          </li>
+          <li>
+            <a href="/schools/">{{ T.navSchools }}</a>
+          </li>
+          <li>
+            <a href="https://blog.omegaup.com">{{ T.navBlog }}</a>
+          </li>
+          <li>
+            <a href="https://omegaup.com/preguntas/">{{ T.navQuestions }}</a>
+          </li>
+        </ul>
+      </div>
+      <div class="footer-list-section">
+        <h4 class="section-title">{{ T.frontPageFooterOrganization }}</h4>
+        <ul>
+          <li>
+            <a href="https://omegaup.org/#about">{{ T.frontPageFooterAboutUs }}</a>
+          </li>
+          <li>
+            <a href="https://omegaup.org/#team">{{ T.frontPageFooterTeam }}</a>
+          </li>
+        </ul>
+      </div>
+      <div class="footer-list-section">
+        <h4 class="section-title">{{ T.frontPageDevelopers }}</h4>
+        <ul>
+          <li>
+            <a href="https://github.com/omegaup/omegaup/wiki/C%C3%B3mo-empezar-a-desarrollar">{{
+            T.frontPageFooterHelpUs }}</a>
+          </li>
+          <li>
+            <a href="https://github.com/omegaup/omegaup">Github</a>
+          </li>
+        </ul>
+      </div>
+      <div class="footer-list-section">
+        <h4 class="section-title">{{ T.frontPageFooterContact }}</h4>
+        <ul>
+          <li>
+            <a href="mailto:hello@omegaup.com">hello@omegaup.com</a>
+          </li>
+        </ul>
+        <div class="social-icons">
+          GITHUB / FACEBOOK
+        </div>
+      </div>
+    </div>
+    <div class="copy">
+      <div class="copy-content">
+        <div class="">
+          Copyright © 2019 OmegaUp. All rights reserved.​
```

Comment:
```suggestion
          Copyright © 2019 omegaUp, Inc. All rights reserved.​
```

@joemmanuel esta es la manera correcta de referirse a la organización?